tools/epanet-output/src/outputapi.h

```diff
@@ -82,51 +82,62 @@ typedef enum {
 #endif
 
 
-ENResultsAPI* DLLEXPORT ENR_init(void);
-
-int DLLEXPORT ENR_open(ENResultsAPI* enrapi, const char* path);
-
-int DLLEXPORT ENR_getVersion(ENResultsAPI* enrapi, int* version);
-
-int DLLEXPORT ENR_getNetSize(ENResultsAPI* enrapi, ENR_ElementCount code, int* count);
-
-int DLLEXPORT ENR_getUnits(ENResultsAPI* enrapi, ENR_Unit code, int* unitFlag);
-
-int DLLEXPORT ENR_getTimes(ENResultsAPI* enrapi, ENR_Time code, int* time);
-
-int DLLEXPORT ENR_getElementName(ENResultsAPI* enrapi, ENR_ElementType type,
-		int elementIndex, char* name);
-
-int DLLEXPORT ENR_getEnergyUsage(ENResultsAPI* enrapi, int pumpIndex,
-		int* linkIndex, float* values);
-int DLLEXPORT ENR_getNetReacts(ENResultsAPI* enrapi, float* values);
-
-float* DLLEXPORT ENR_newOutValueSeries(ENResultsAPI* enrapi, int startPeriod,
-        int endPeriod, int* length, int* errcode);
-float* DLLEXPORT ENR_newOutValueArray(ENResultsAPI* enrapi, ENR_ApiFunction func,
-        ENR_ElementType type, int* length, int* errcode);
-
-int DLLEXPORT ENR_getNodeSeries(ENResultsAPI* enrapi, int nodeIndex, ENR_NodeAttribute attr,
-        int startPeriod, int length, float* outValueSeries);
-int DLLEXPORT ENR_getLinkSeries(ENResultsAPI* enrapi, int linkIndex, ENR_LinkAttribute attr,
-        int startPeriod, int length, float* outValueSeries);
-
-int DLLEXPORT ENR_getNodeAttribute(ENResultsAPI* enrapi, int periodIndex,
-        ENR_NodeAttribute attr, float* outValueArray, int* length);
-int DLLEXPORT ENR_getLinkAttribute(ENResultsAPI* enrapi, int periodIndex,
-        ENR_LinkAttribute attr, float* outValueArray, int* length);
-
-int DLLEXPORT ENR_getNodeResult(ENResultsAPI* enrapi, int periodIndex, int nodeIndex,
-        float* outValueArray);
-int DLLEXPORT ENR_getLinkResult(ENResultsAPI* enrapi, int periodIndex, int linkIndex,
-        float* outValueArray);
-
-int DLLEXPORT ENR_free(float *array);
-
-int DLLEXPORT ENR_close(ENResultsAPI* enrapi);
-
-int DLLEXPORT ENR_errMessage(int errcode, char* errmsg, int n);
-
+int DLLEXPORT ENR_init(ENR_Handle* p_handle_out);
+// wrapped
```

Comment:
should prob. remove the "wrapped" notes - these comments don't add important information. although if a function is not wrapped, then that might be pertinent info.

Reply:
Oops. I will make this change as well.